server/boot/controller-boot.js

```diff
@@ -0,0 +1,30 @@
+var controller;
+
+module.exports = function(app) {
+  // here app.controller(ctrl) creates a controller wrapper to register
+  // the 'ctrl' class with strong-remoting
+  app.controller(ctrl).remoteMethod('speak', {
+    http: { path: '/', verb: 'get' },
+    returns: { arg: 'msg', type: 'string', root: true },
+  });
+};
+
+function ctrl(app, ctx) {
+  // actually ends up registering at /api/ctrl
+  this.name = 'TestName';
```

Comment:
> `ctrl.name = 'MyControllerName'`

IIRC, you cannot override function name like that.

```
$ node
> var fn = function ctrl() {};
undefined
> fn.name
'ctrl'
> fn.name = 'TestName';
'TestName'
> fn.name
'ctrl'
```

For models, we use the property `modelName`, I guess we could use `controllerName` for controllers?

The more important question to answer is why would you like to set the controller name in the first place, I mean what do you use the name for?